mmd_tools/operators/misc.py

```diff
@@ -50,27 +50,35 @@ def execute(self, context):
         obj = context.active_object
         root = mmd_model.Model.findRoot(obj)
         if root is None:
-            self.report({ 'ERROR' }, 'Select a MMD model') 
-            return { 'CANCELLED' } 
+            # self.report({ 'ERROR' }, 'Select a MMD model') 
+            # return { 'CANCELLED' }
+            # find all meshes in the active layer
+            act_layer = bpy.context.scene.active_layer
+            meshes_list = [o for o in bpy.data.objects
+                           if o.layers[act_layer] and o.type == 'MESH' and o.mmd_type == 'NONE']
+            active_mesh = meshes_list[0]
```

Comment:
The idea is to allow existing Blender meshes to be quickly added to mmd_root. When modelling is very frequent to see models composed of different meshes. Attaching all the meshes to the mmd_root can be troublesome. With this you can join all your meshes and attach the resulting meshes to the mmd_root armature. Another possibility could be to create another operator for that purpose (finding existing meshes and attaching them to mmd_root)